### ISSUE TITLE
Reduce confusion between the Type and Resource Type facets

### ISSUE THREAD

**Issue by sandbergja** [lbcclib/community_archive#26]
We have two facets with very similar names:

- Type (which is the Work type, like OER, Newspaper Issue, Syllabus, etc.)
- Resource Type (which is the OER Commons type, mainly available only for OER works)

Users get them confused.  Another issue is that there are similarities in the controlled vocabs for each facet.  The results for Type= Newspaper Issue is very different from the results for Resource Type = Newspaper.

I find a lot of value in the Type facet.  I don't find much value in displaying the Resource Type facet; @mwillihooper: do you use that in your OER work?

I'd like to propose not displaying the Resource Type facet on the search results page.  We would still display those data on the Work page and in OAI-PMH, but I don't think that they are confusing in those places.

FWIW, it looks like some other hyrax implementations go the other way: hiding the Type facet and keeping the Resource type facet.  I don't think that's a good option for us, since we have better data in the Type facet.

**Comment by sandbergja** [comment]
Plan: bring the following values from the Type facet into the Resource Type facet, using the custom indexers at https://github.com/lbcclib/community_archive/tree/master/app/indexers:
* Image
* Newspaper Issue
* Syllabus, which would include:
  * Works with the Syllabus Work Type
  * Works with the OER Work Type, but where the uploader specifically chose Syllabus from the OERC vocabulary
* Video

And then we could turn off the Type facet.

This would mean we would no longer be able to use the facets to narrow down to:
* Collection
* Document
* Open Educational Resources (we could just use the collection instead)

**Comment by sandbergja** [comment]
I pushed the code; just waiting on everything to be reindexed now

**Comment by sandbergja** [comment]
The re-index is finished!  These two facets are now combined.  Hopefully this is much less confusing to our users.
